docs/Reference/Collection/Indexes.md

```diff
@@ -53,7 +62,7 @@ const client = new jsc8({url: "https://gdn1.macrometa.io", token: "XXXX"});
 //---- OR ----
 const client = new jsc8({url: "https://gdn1.macrometa.io", apiKey: "XXXX"});
 
-const hashIndex = await client.addHashIndex("some-collection", 'favorite-color');
+const hashIndex = await client.addHashIndex("some-collection", ['field1'], { unique: true, sparse: true, deduplicate: true });
```

Comment:
replace field1 with `field-name` or add field2 as well to the array.

Reply:
Do it for all instances

Reply:
updated